[2022-10-29T01:17:21.519+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: ENG_hourly_process_v1.get_hourly_demand scheduled__2022-10-29T00:05:00+00:00 [queued]>
[2022-10-29T01:17:21.532+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: ENG_hourly_process_v1.get_hourly_demand scheduled__2022-10-29T00:05:00+00:00 [queued]>
[2022-10-29T01:17:21.533+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-29T01:17:21.533+0000] {taskinstance.py:1363} INFO - Starting attempt 3 of 4
[2022-10-29T01:17:21.533+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-29T01:17:21.554+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): get_hourly_demand> on 2022-10-29 00:05:00+00:00
[2022-10-29T01:17:21.561+0000] {standard_task_runner.py:55} INFO - Started process 2725 to run task
[2022-10-29T01:17:21.564+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'ENG_hourly_process_v1', 'get_hourly_demand', 'scheduled__2022-10-29T00:05:00+00:00', '--job-id', '150', '--raw', '--subdir', 'DAGS_FOLDER/ENG_hourly_process_dag.py', '--cfg-path', '/tmp/tmp82fk1nas']
[2022-10-29T01:17:21.566+0000] {standard_task_runner.py:83} INFO - Job 150: Subtask get_hourly_demand
[2022-10-29T01:17:21.641+0000] {task_command.py:376} INFO - Running <TaskInstance: ENG_hourly_process_v1.get_hourly_demand scheduled__2022-10-29T00:05:00+00:00 [running]> on host 6800cd8e39a3
[2022-10-29T01:17:21.739+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=colmo786@gmail.com
AIRFLOW_CTX_DAG_OWNER=energy
AIRFLOW_CTX_DAG_ID=ENG_hourly_process_v1
AIRFLOW_CTX_TASK_ID=get_hourly_demand
AIRFLOW_CTX_EXECUTION_DATE=2022-10-29T00:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-29T00:05:00+00:00
[2022-10-29T01:17:21.748+0000] {logging_mixin.py:120} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:297 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-10-29T01:17:21.756+0000] {logging_mixin.py:120} INFO - INFO Module build_postgres_cnxn: DB Connection to host local_pgdb Ok
[2022-10-29T01:17:22.007+0000] {logging_mixin.py:120} INFO - INFO api_request_to_pandas: API request executed Ok. Number of records returned: 0
[2022-10-29T01:17:22.007+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ENG_hourly_process_dag.py", line 49, in get_hourly_demand
    , user=user, password=password, port=port, verbose=True)
  File "/opt/airflow/dags/energy.py", line 161, in _get_hourly_demand
    df_demand.fecha = pd.to_datetime(df_demand.fecha.astype(str).str[:19], format='%Y-%m-%d %H:%M:%S')
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/generic.py", line 5487, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'fecha'
[2022-10-29T01:17:22.020+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=ENG_hourly_process_v1, task_id=get_hourly_demand, execution_date=20221029T000500, start_date=20221029T011721, end_date=20221029T011722
[2022-10-29T01:17:22.036+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 150 for task get_hourly_demand ('DataFrame' object has no attribute 'fecha'; 2725)
[2022-10-29T01:17:22.057+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-10-29T01:17:22.092+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
